### DISCUSSION TITLE
Possibility of "reclaiming" some top space when thumbnail is deactivated

### DISCUSSION THREAD

**Discussion by jsonnentag** [rrirower/DP2SceneGraphWizard#79] [Q&A]
What is the possibility of reclaiming the empty space in the top right part of the screen (where the enlarged thumbnail image can now be easily turned off with a switch in the manifest) so that video descriptions can have a lot more area to appear in?

**Comment by rrirower** [comment]
I'd have to investigate it further.  This would be marked as a low priority.
